tests/tests.sh

```diff
@@ -12,6 +12,9 @@ function check_output_status() {
 
 cd dev_utils || exit 1
 
+token="$(bash keys/sign_jwt.sh ES256 /keys/jwt.key)"
+sed -i 's/TOKEN/$token/' proxyS3
```

Comment:
```suggestion
sed -i "s/TOKEN/$token/" proxyS3
```

I believe we want `$token` to be expanded here.